src/cosmic_ray/operators/provider.py

```diff
@@ -37,7 +37,7 @@ def __iter__(self):
         return iter(_OPERATORS)
 
     def __getitem__(self, name):
-        if name == 'no_op':
+        if name == 'NoOp':
```

Comment:
This seems like a separate change.

Reply:
It's just a bug correction: baseline claims for 'NoOp' and the naming convention for operator  names is CamelCase (coming from class name convention).